int32 num_robots
int32 num_boxes
geometry_msgs/Point[] initial_robot
geometry_msgs/Point[] initial_box
geometry_msgs/Point[] final_box
